homework-submission/W1-homework/index.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+/**
+ * A function component that inserts a static item in a list
+ * @param {props} param0 The destructured properties description and deadline
+ * @return List item with the values of the properties
+ */
+const StaticTodoItem = ({ description, deadline }) => {
+  return (
+    <li>
+      <h3>{description}</h3>
+      <span>{deadline}</span>
+    </li>
+  );
+};
+
+/**
+ * A function component that inserts a dynamic item in a list
+ * @param {props} param0 The array of todoItems
+ * @return List item with the values of the todoItems of the array if todoItem is not done yet (false)
+ * @return List item with the values of the todoItems of the array if todoItem is done (true)
+ * @return The dynamic list of todoItems
+ */
+const DynamicTodoItem = ({ todoItems }) => {
+  const items = todoItems.map(({ id, description, deadline, done }) => {
+    if (!done) {
+      return (
+        <li key={id}>
+          <h3>{description}</h3>
+          <span>{deadline}</span>
+        </li>
+      );
+    }
+    return (
```

Comment:
you are repeating your self here too, if all of the return content only for the `className` you could consider using ternary operator on line 28 in the `className` instead of this return and the `if` statement as the following:
```javascript
className={done ? "todo-item-done" : ""}
```

And that's it:)

Reply:
Yes, you are right... I've tried both ways, but this one is better.